packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -192,6 +192,63 @@ describe('CDK Include', () => {
     );
   });
 
+  test('can ingest a template with Fn::Sub in string form with escaped and unescaped references and output it unchanged', () => {
+    includeTestTemplate(stack, 'fn-sub-valid-string.json');
+
+    expect(stack).toMatchTemplate(
+      loadTestFileToJsObject('fn-sub-valid-string.json'),
+    );
+  });
+
+  test('can ingest a template with Fn::Sub in string form with escaped references and output it unchanged', () => {
```

Comment:
Redundant with above...? Above validates `escaped and unescaped references`, this one validates `escaped references`...?

Reply:
This is a poor name for this test, it's supposed to check the edge case in parsing the escaped references.